Patches/ChatCommandPatch.cs

```diff
@@ -845,7 +845,7 @@ public static void SendRolesInfo(string role, byte playerId, bool isDev = false,
                 var settings = new StringBuilder();
                 if (Options.CustomRoleSpawnChances.ContainsKey(rl))
                 {
-                    settings.AppendLine($"<size=70%><u>Settings for <{Main.roleColors[rl]}>{roleName}</color>:</u>");
+                    settings.AppendLine($"<size=70%><u>{GetString("SettingsForRoleText")}</u> <{Main.roleColors[rl]}>{roleName}</color>:");

```

Comment:
Move </u> back lol

Reply:
![image](https://github.com/Gurge44/TOHE_PLUS/assets/95672691/571f760e-5ca4-41d4-ba0e-640dfdec387c)
what? </u> is here already